handsontable/src/dataMap/metaManager/metaSchema.js

```diff
@@ -1840,6 +1840,46 @@ export default () => {
      */
     disableVisualSelection: false,
 
+    /**
+     * @description
+     * The `dialog` option configures the [`Dialog`](@/api/dialog.md) plugin.
+     *
+     * You can set the `dialog` option to one of the following:
+     *
+     * | Setting   | Description                                                                 |
+     * | --------- | --------------------------------------------------------------------------- |
+     * | `false`   | Disable the [`Dialog`](@/api/dialog.md) plugin                           |
+     * | `true`    | Enable the [`Dialog`](@/api/dialog.md) plugin with default options     |
+     * | An object | Enable the [`Dialog`](@/api/dialog.md) plugin with custom configuration |
```

Comment:
It’s worth to extend the docs for "additional options" section, for example https://github.com/handsontable/handsontable/blob/develop/handsontable/src/dataMap/metaManager/metaSchema.js#L3596